code/modules/projectiles/projectile/ricochet.dm

```diff
@@ -385,3 +385,27 @@
 	spawn(30)
 		qdel(src)
 
+
+
+/obj/item/projectile/ricochet/taser
+	name = "electrode"
+	nodamage = 1
+	stun = 10
+	weaken = 10
+	stutter = 10
+	jittery = 20
+	agony = 10
+	hitsound = 'sound/weapons/taserhit.ogg'
+
+/obj/item/projectile/ricochet/taser/hit_apply(var/mob/living/secoff, var/blocked = 0)
```

Comment:
This copypasted implementation of hit_apply doesn't account for manifested ghosts, which the other implementation did.
Re-use the proc instead of copypasting.